config/onprem-iso-config.ign

```diff
@@ -37,7 +37,7 @@
         "path": "/etc/assisted-service/auth.json",
         "filesystem": "root",
         "contents": {
-          "source": "data:;base64,replace-with-your-base64-encoded-pull-secret"
+          "source": "data:;base64,Y2hhbmdlbWUK"
```

Comment:
Is this on purpose?

Reply:
It is on purpose. "replace-with-your-base64-encoded-pull-secret" is not base64, so I had to use a base64 string there.

Reply:
```bash
➜  ~ python -c "import base64; print(base64.decodebytes(b'Y2hhbmdlbWUK'))"
b'changeme\n'
```

Why do we need it to be an actual base64? I expect it to be clear it is not a valid value

Reply:
I've found a different way to do this that removes the need to base64 encode. I think it will be clearer now.